js-intro.js

```diff
@@ -11,14 +11,29 @@ var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
+var checkBoiling = (temp) => {
+    if( temp === 212) return `212 is above boiling point`
+    else if(temp > 212) return `${temp} is above boiling point`
+    else return `${temp} is belove boiling point`
+}
 
+console.log(checkBoiling(temp1))
+console.log(checkBoiling(temp2))
+console.log(checkBoiling(temp3))
 
 
 // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
 
 var myNumbers1 = [3, 7, 0, 6, -9]
 
+var mult5 = (nums) => {
+    for(let i = 0; i < nums.length; i++){
+        nums[i] = nums[i] * 5
```

Comment:
Cool solution without creating a new array :) I feel like you know what your intention was here but you might want to consider not altering the original array in case you need the original data later.